src/pixel.js

```diff
@@ -10,27 +10,35 @@ const TAG = 'pixel';
  * @param {!Window} win
  * @param {string} src
  * @param {?string=} referrerPolicy
+ * @param {?boolean=} attributionReportingEligible
+ * @param {string=} attributionsrc
  * @return {!Element}
  */
-export function createPixel(win, src, referrerPolicy) {
+export function createPixel(win, src, referrerPolicy,
+                             attributionReportingEligible,
+                             attributionSrc) {
   // Caller need to verify window is not destroyed when creating pixel
   if (referrerPolicy && referrerPolicy !== 'no-referrer') {
     user().error(TAG, 'Unsupported referrerPolicy: %s', referrerPolicy);
   }
 
   return referrerPolicy === 'no-referrer'
-    ? createNoReferrerPixel(win, src)
-    : createImagePixel(win, src);
+    ? createNoReferrerPixel(win, src, attributionReportingEligible, attributionSrc)
+    : createImagePixel(win, src, false, attributionReportingEligible, attributionSrc);
 }
 
 /**
  * @param {!Window} win
  * @param {string} src
+ * @param {?boolean=} attributionReportingEligible
+ * @param {string=} attributionsrc
  * @return {!Element}
  */
-function createNoReferrerPixel(win, src) {
+function createNoReferrerPixel(win, src,
+                                attributionReportingEligible,
+                                attributionSrc) {
   if (isReferrerPolicySupported()) {
-    return createImagePixel(win, src, true);
+    return createImagePixel(win, src, true, attributionReportingEligible, attributionSrc);
   } else {
```

Comment:
note I think we only get here on ie11 which we dont actually support anymore

Reply:
Do you mean the else condition where referrer policy is not supported? Is this confirmed? I just remembered we have to add allow='attributing-reporting' for img tags generated on iframes. But this is the only place we generate iframes for pixels, so if this is only ie11, we don't have to add it and can even remove adding attributionSrc in this condition as we only care about Chrome browers.

Reply:
https://caniuse.com/mdn-html_elements_img_referrerpolicy seems if all we care about is chrome we are safe.

Reply:
Nice, I removed the attributionsrc parameter in the non-referrer case.